galaxy_ng/app/api/ui/views/landing_page.py

```diff
@@ -0,0 +1,161 @@
+from rest_framework.permissions import IsAuthenticatedOrReadOnly
+from rest_framework.response import Response
+from pulp_ansible.app.models import (
+    CollectionVersion,
+    AnsibleDistribution
+)
+from galaxy_ng.app.models import Namespace
+from django.conf import settings
+from galaxy_ng.app.api import base as api_base
+
+golden_name = settings.GALAXY_API_DEFAULT_DISTRIBUTION_BASE_PATH
+
+
+class LandingPageView(api_base.APIView):
+    permission_classes = [IsAuthenticatedOrReadOnly]
```

Comment:
You should use an access policy for this. Docs on RBAC can be found here: https://github.com/ansible/galaxy_ng/wiki/RBAC-in-Galaxy-NG

Access policies are declared here: https://github.com/ansible/galaxy_ng/blob/master/galaxy_ng/app/access_control/access_policy.py

Policy statements are declared here: https://github.com/ansible/galaxy_ng/blob/master/galaxy_ng/app/access_control/statements/insights.py

The statement for this can look like this. This will grant permissions on users that have the correct Red Hat entitlements.

```
        {
            "action": ["retrieve"],
            "principal": "authenticated",
            "effect": "allow",
            "condition": "has_rh_entitlements",
        },
```

Reply:
@newswangerd I am not sure how to do this, this is a view and not a viewset, and as it is currently it has to have the `permission_classes` which does not accepts this.

Reply:
@brumik you set the access policy class as the `permission_classes`. Here's an example on one of the views for execution environments: https://github.com/ansible/galaxy_ng/blob/master/galaxy_ng/app/api/ui/views/index_execution_environments.py#L22

That should be a good example to follow